plasma_framework/contracts/src/vaults/verifiers/Erc20DepositVerifier.sol

```diff
@@ -12,9 +12,14 @@ import {PaymentOutputModel as DepositOutputModel} from "../../transactions/outpu
 contract Erc20DepositVerifier is IErc20DepositVerifier {
     using DepositOutputModel for DepositOutputModel.Output;
 
+    uint256 public supportedOutputType;
     // Hardcoded transaction type for payment transaction
     uint8 constant private DEPOSIT_TX_TYPE = 1;
 
+    constructor(uint256 outputType) public {
```

Comment:
If we are doing this, might consider to just inject `txType` too.
I am fine with hardcode both or inject both, but better to be consistent

Reply:
done